### PR TITLE
Create Head-of-Product.md

### PR DESCRIPTION
**Pull request by guypod**
WIP

### CODE REVIEW

File: Head-of-Product.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## About [Snyk](https://snyk.io)
+
+<img src="https://snyk.io/images/snyk-dog.png" width="100" alt="Snyk logo" />
+
+Snyk's mission is to help developers use open source code and stay secure. 
+Open Source code, pulled from npm, Maven, RubyGems and others, is often the majority of code in an application, and yet it receives no security attention. 
+Most developers do nothing to track known vulnerabilities in the packages they use, and trust this code as though its their own. 
+This risk is massive today, and is only made worse by growing use of packages and trends such as Serverless and PaaS, who secure the server's dependencies but leave the app to fend with its dependencies security alone.
```
Comment (geva) [inline Head-of-Product.md]: Suggest to rephrase so the note on Serverless and PaaS is more digestible to someone who's not entrenched in the industry:
"This risk is massive today, and is only made worse by the growing use of packages and the simplicity offered by package managers. This is compounded by the fact that unpatched servers, which are the target of many hackers, are slowly being abstracted away by PaaS and Serverless offerings, making vulnerabilities in open source code take center stage"

File: Head-of-Product.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+At Snyk we believe this problem can only be fixed by developers and DevOps teams, and built into the way we create software. 
+We build a SaaS CI/CD and runtime monitoring product that helps organisations continuosly find and fix vulnerable dependencies without slowing down.
```
Comment (geva) [inline Head-of-Product.md]: continuously

Comment (geva) [inline Head-of-Product.md]: I don't believe 'runtime' would be understood like you mean it to by someone not from within

Comment (geva) [inline Head-of-Product.md]: I find writing: "find, fix (and prevent!)" makes prevention visually stand out

File: Head-of-Product.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+It's important to understand Snyk is a dev tooling company at heart. 
+We care about securing the open source world itself as much as as we care about our enterprise grade solution that let large companies consume open source code while controlling risk. 
+We receive regular community contributions (and love!), have more than 600 badges in GitHub championing us, and continuously protect over 10,000 open source repositories.
```
Comment (geva) [inline Head-of-Product.md]: 600000 downloads is another public metric, that is way more impressive than 600 badges

File: Head-of-Product.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+Snyk's mission is to help developers use open source code and stay secure. 
+Open Source code, pulled from npm, Maven, RubyGems and others, is often the majority of code in an application, and yet it receives no security attention. 
+Most developers do nothing to track known vulnerabilities in the packages they use, and trust this code as though its their own. 
+This risk is massive today, and is only made worse by growing use of packages and trends such as Serverless and PaaS, who secure the server's dependencies but leave the app to fend with its dependencies security alone.
+
+At Snyk we believe this problem can only be fixed by developers and DevOps teams, and built into the way we create software. 
+We build a SaaS CI/CD and runtime monitoring product that helps organisations continuosly find and fix vulnerable dependencies without slowing down.
+
+It's important to understand Snyk is a dev tooling company at heart. 
+We care about securing the open source world itself as much as as we care about our enterprise grade solution that let large companies consume open source code while controlling risk.
```
Comment (geva) [inline Head-of-Product.md]: let --> lets

File: Head-of-Product.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+## Head of Product - London or Israel
+
+We are looking for a senior head of product, based in one of our main offices in London or Tel Aviv. 
+
+In this role, you will own all product management activities, translating our vision and user feedback into product needs and working closely with engineering to make it a reality. You will also work regularly with customers and prospects, balancing high-touch enterprise customer interactions with understanding the many thousands of users who use your product self serve.
+
+As Snyk grows, you'll have the opportunity to build, lead and grow a product team, scaling up as our company hits its next growth stages. You will be a part of the company's leadership, and will influence practically all parts of the company. You will be reporting directly to the CEO.
+
+
+### Responsibilities
```
Comment (geva) [inline Head-of-Product.md]: This whole section reads fluently, but content seems pretty "product management generic"

File: Head-of-Product.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+#### Required
+- 10+ years of industry experience 
+- 3+ years of product management experience
+- Deep technical understanding, which allow a 
+- Experience building or managing at least one of Security, Developer or DevOps products
+- Excellent written and verbal communication skills in English
+- Willingness to regularly travel between our Israel and London offices (roughly every 6 weeks)
+
+#### Nice-to-have
```
Comment (geva) [inline Head-of-Product.md]: Familiarity with open source products/ecosystems

File: Head-of-Product.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+### Skills & experience
+
+#### Required
+- 10+ years of industry experience 
+- 3+ years of product management experience
+- Deep technical understanding, which allow for a good understanding of our solution and users
+- Experience building or managing at least one of Security, Developer or DevOps products
+- Excellent written and verbal communication skills in English
+- Willingness to regularly travel between our Israel and London offices (roughly every 6 weeks)
```
Comment (maban) [inline Head-of-Product.md]: Israel and London > Tel Aviv and London (for consistency)

File: Head-of-Product.md (hunk 5 above)
Comment (guypod) [inline Head-of-Product.md]: Tough to make it non-generic... I think the previous pieces explain why it's special.

**Pull request merged**